src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/retry/delay/impl/RetryConsumerDelayStrategy.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.companieshouse.chipsrestinterfacesconsumer.retry.delay.impl;
+
+import org.springframework.beans.factory.annotation.Value;
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.common.ApplicationLogger;
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.retry.delay.ConsumerDelayStrategy;
+
+import java.util.HashMap;
```

Comment:
I think this is now an unused import.
In intellij you can right click 'src' and click "Optimise imports" and it will do this for your entire project.
I try my best to do that just before pushing commits

Reply:
I think that's gone now.